packages/@o3r/core/schematics/rule-factories/customization-environment/index.ts

```diff
@@ -100,7 +100,6 @@ export function updateCustomizationEnvironment(rootPath: string, o3rCoreVersion?
     updatedRecorder = insertImportToModuleFile(updatedRecorder, 'initializeCustomProviders', '../customization/custom-providers.empty');
 
     updatedRecorder = addToNgModuleMetadata(updatedRecorder, 'imports', '...entry.customComponentsModules');
-    updatedRecorder = addToNgModuleMetadata(updatedRecorder, 'entryComponents', '...entry.customComponents');
```

Comment:
What about standalone component that doesn't have module ?

Reply:
Here, we're updating the `app.module` to add support for the customization environment. 
Entry components were deprecated in Angular 9 but in Angular 16 were completely removed from the types so the goal here is to remove any reference to it.

Reply:
How do we handle the component replacement, now ?

Reply:
by importing their modules (line 102)